packages/ripple/index.tsx

```diff
@@ -71,12 +72,12 @@ export interface RippledComponentInterface<Surface, Activator = Element> {
   handleKeyDown: React.KeyboardEventHandler<Surface>;
   handleKeyUp: React.KeyboardEventHandler<Surface>;
   activateRipple: (e: ActivateEventTypes<Surface>) => void;
-  deactivateRipple: (e: ActivateEventTypes<Surface>) => void;
+  deactivateRipple: () => void;
   classes: string;
   style: React.CSSProperties;
   displayName: string;
   createAdapter: (surface: Surface, activator?: Activator) => MDCRippleAdapter;
-  updateCssVariable: (varName: keyof React.CSSProperties, value: string | number) => void;
+  updateCssVariable: (varName: string, value: string | null) => void;
```

Comment:
I think you can revert the keyof React.CSSProperties.

Reply:
MDC-web has string in their interface and when I limit it to CSS properties type verification fails. Is there any better solution? And technically it is css property, which can be different with react css properties (borderBottom vs border-bottom, for example)

Reply:
ah yes you're right...ok this seems like the correct change